src/components/Article/Article.test.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { render, cleanup } from '@testing-library/react';
+import Article from './Article';
+
+describe('Article component', () => {
+  afterEach(() => cleanup());
+  it('renders Article', () => {
+    const { asFragment } = render(<Article />);
```

Comment:
Although it passes without them, it's a good idea to pass your props to the component in the test